### PR TITLE
Add Base/B3 Sepolia Testnets.

### PR DESCRIPTION
**Pull request by sentilesdal**
Adding support for Base Sepolia and B3 Sepolia testnets.

### CODE REVIEW

File: apps/hyperdrive-trading/src/network/b3Sepolia.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { defineChain } from "viem";
+import { chainConfig } from "viem/op-stack";
+
+export const sourceId = 1155111;
+export const b3Sepolia = /*#__PURE__*/ defineChain({
```
Comment (DannyDelott) [inline apps/hyperdrive-trading/src/network/b3Sepolia.ts:5]: Couple thoughts:
- Is the `sourceId` correct (should it be `1993`? Maybe add a comment) and does it even need to be exported?
- Can we remove the `#__PURE__`?
- Let's make `cloudChain.ts` consistent with the pattern here, I like the `defineChain` method you found 👍

File: apps/hyperdrive-trading/src/ui/appconfig/useAppConfig.ts [hunk 2]
```diff
@@ -5,10 +5,31 @@ import {
...
+  tags: [],
+  registryAddress: ZERO_ADDRESS,
+  hyperdrives: [],
+  tokens: [],
+  protocols: {},
+};
+
+const baseSepoliaAppConfig: AppConfig = {
+  ...emptyAppConfig,
+  chainId: 84532,
```
Comment (DannyDelott) [inline apps/hyperdrive-trading/src/ui/appconfig/useAppConfig.ts:25]: let's pull re-use from `baseSepolia.id`.

File: apps/hyperdrive-trading/src/ui/appconfig/useAppConfig.ts [hunk 3]
```diff
@@ -5,10 +5,31 @@ import {
...
+};
+
+const b3SepoliaAppConfig: AppConfig = {
+  ...emptyAppConfig,
+  chainId: 1993,
```
Comment (DannyDelott) [inline apps/hyperdrive-trading/src/ui/appconfig/useAppConfig.ts:30]: same thing here: `b3Sepolia.id`

File: turbo.json [hunk 4]
```diff
@@ -2,15 +2,28 @@
   "$schema": "https://turborepo.org/schema.json",
   "tasks": {
     "build:crates": {
-      "dependsOn": ["^build"],
-      "outputs": ["packages/hyperdrive-wasm/**", "packages/fixed-point-wasm/**"]
+      "dependsOn": [
```
Comment (DannyDelott) [inline turbo.json:5]: Did you intend to change this file?

File: apps/hyperdrive-trading/src/network/wagmiClient.ts [hunk 5]
```diff
@@ -9,18 +9,20 @@ import {
...
 
 const {
   VITE_LOCALHOST_NODE_RPC_URL,
   VITE_CUSTOM_CHAIN_NODE_RPC_URL,
-  VITE_CUSTOM_CHAIN_ADDRESSES_URL,
   VITE_CUSTOM_CHAIN_CHAIN_ID,
   VITE_WALLET_CONNECT_PROJECT_ID,
   VITE_SEPOLIA_RPC_URL,
+  VITE_BASE_SEPOLIA_RPC_URL,
+  VITE_B3_SEPOLIA_RPC_URL,
```
Comment (DannyDelott) [inline apps/hyperdrive-trading/src/network/wagmiClient.ts:25]: Have we added these to the `.env.sample`?

File: turbo.json (hunk 4 above)
Comment (sentilesdal) [reply turbo.json:5]: it was just auto-formatted, the only change is to `env` where I added:
```
        "VITE_BASE_SEPOLIA_RPC_URL",
        "VITE_B3_SEPOLIA_RPC_URL",
        ```

File: apps/hyperdrive-trading/src/network/wagmiClient.ts (hunk 5 above)
Comment (sentilesdal) [reply apps/hyperdrive-trading/src/network/wagmiClient.ts:25]: just added

File: apps/hyperdrive-trading/src/network/b3Sepolia.ts (hunk 1 above)
Comment (sentilesdal) [reply apps/hyperdrive-trading/src/network/b3Sepolia.ts:5]: sourceId is supposed to be Sepolia, I followed prior art in viem's baseSepolia.ts file.  Weirdly its actually wrong in that file, they have 11_55_111 and it should be 111_55_111.  our b3Sepolia file now has the right one.

Comment (sentilesdal) [reply apps/hyperdrive-trading/src/network/b3Sepolia.ts:5]: here's the file if you want to check it out:
`hyperdrive-frontend/node_modules/viem/chains/definitions/baseSepolia.ts`

Comment (sentilesdal) [reply apps/hyperdrive-trading/src/network/b3Sepolia.ts:5]: I updated cloudChain.ts to use defineChain.

**Review by DannyDelott — APPROVED** [review]

**Pull request merged**